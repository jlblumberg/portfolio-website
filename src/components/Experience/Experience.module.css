.flexContainer { 
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  height: 100vh;
  background-color: rgb(245, 245, 245);
}

.headerText {
  max-width: 90%;
  margin-bottom: 3%;
}

.descriptionText {
  text-align: center;
  max-width: 70%;
}