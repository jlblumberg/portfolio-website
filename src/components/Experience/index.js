import React from 'react';
import styles from './Experience.module.css';
import { Typography, IconButton } from '@material-ui/core';
import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';

const Experience = () => {

  const handleClick = () => {
    alert('add in scroll functionality later')
  }

  return (
    <div className={styles.flexContainer}>
      <div className={styles.headerText} id='header'>
        <Typography variant='h5' color='textSecondary'>EXPERIENCE</Typography>
      </div>
      <div className={styles.descriptionText} id='description'>
        <Typography variant='h6'>
          <p>
            experience descriptions
          </p>
        </Typography>
      </div>
      <div>
        <IconButton color='primary' id='down-button' onClick={handleClick}>
          <KeyboardArrowDownIcon fontSize='large' />
        </IconButton>
      </div>
    </div>
  );
}

export default Experience;