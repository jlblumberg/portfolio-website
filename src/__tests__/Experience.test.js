import React from 'react';
import { shallow } from 'enzyme';
import Experience from '../components/Experience';

describe('Experience', () => {

  const wrapper = shallow(<Experience />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has a down button which takes you to the work section', () => {
    expect(wrapper.find('#down-button').exists()).toBe(true);
  });

});
