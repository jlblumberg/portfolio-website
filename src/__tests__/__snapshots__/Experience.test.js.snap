// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience renders correctly 1`] = `ShallowWrapper {}`;
